setup.py

```diff
@@ -77,7 +57,17 @@ def get_packages(package):
         'Programming Language :: Python :: 3.4',
         'Topic :: Utilities',
     ],
+    # install_requires=['cryptography'],
     test_suite='tests',
-    tests_require=['pytest', 'pytest-cov'],
-    cmdclass={'test': PyTest},
```

Comment:
Need to look into that - perhaps switching to the more explicit setup.py pytest is acceptable?